src/agent/configuration_parser/tests/configuration_parser_test.cpp

```diff
@@ -89,7 +90,7 @@ TEST(ConfigurationParser, GetConfigString)
           string_conf: string
     )";
     const auto parserStr = std::make_unique<configuration::ConfigurationParser>(strConfig);
-    const auto ret = parserStr->GetConfig<std::string>("agent", "server_url");
+    const auto ret = parserStr->GetConfig<std::string>("agent", "server_url").value_or("192.168.0.11");
```

Comment:
For these tests it may be better to set a different default value than the expected so they validate that the config is read.